pkgs/tools/security/beyond-identity/default.nix

```
@@ -0,0 +1,99 @@
+{ lib, stdenv, fetchurl, dpkg, buildFHSUserEnv
+, glibc, glib, openssl, tpm2-tss
+, gtk3, gnome, polkit, polkit_gnome
+}:
+
+let
+  version = "2.45.0-0";
+  name = "beyond-identity";
+
+  src = fetchurl {
+    url = "https://packages.beyondidentity.com/public/linux-authenticator/deb/ubuntu/pool/focal/main/b/be/${name}_${version}/${name}_${version}_amd64.deb";
+    sha512 = "852689d473b7538cdca60d264295f39972491b5505accad897fd924504189f0a6d8b6481cc0520ee762d4642e0f4fd664a03b5741f9ea513ec46ab16b05158f2";
+  };
+
+  libPath = lib.makeLibraryPath ([
+    glib glibc openssl tpm2-tss
+    gtk3 gnome.gnome-keyring
+    polkit polkit_gnome
+  ]);
+in let beyond-identity = stdenv.mkDerivation {
+  pname = name;
+
+  inherit version;
+  inherit src;
```

```suggestion
  inherit pname  version src;
```
We would also need to adjust this and the inherits can be combined.